docs/enterprise/install/helm.md

```diff
@@ -112,17 +112,17 @@ This setup assumes an existing certificate solution using cert-manager, as descr
      pullPassword: your-access-key
    ```
 
-1. Add the Pomerium Enterprise repository to your Helm configuration:
+1. Add the Pomerium repository to your Helm configuration:
 
    ```bash
-   helm repo add pomerium-enterprise https://releases.pomerium.com
+   helm repo add pomerium https://helm.pomerium.io
```

Comment:
Do we even need this section anymore? This doc lists the Pomerium on Helm doc as a prerequisite, so they should already have the repository in their Helm config.

If you still want to keep it, I'll amend it to the tune of "...if you don't already have it from the OSS install"

Reply:
> Do we even need this section anymore? This doc lists the Pomerium on Helm doc as a prerequisite, so they should already have the repository in their Helm config.

Fair question - I think it's good to have complete instructions.  The repo setup for helm is local only so, if you ran this on a different workstation or workflow, you may not have the repo installed.  
 
> If you still want to keep it, I'll amend it to the tune of "...if you don't already have it from the OSS install"

Something like that would be fine 👍